src/dbos-executor.ts

```diff
@@ -423,8 +431,23 @@ export class DBOSExecutor {
 
     const runWorkflow = async () => {
       let result: R;
+      // Track concurrent executions of the WF
+      if (!this.workflowStats.has(wf.name)) {
+        this.workflowStats.set(wf.name, new WorkflowStats());
+      }
+      const wfs = this.workflowStats.get(wf.name)!;
+      wfs.currentlyExecuting++;
+      wfs.nStarted++;
+
       // Execute the workflow.
       try {
+        // If this is a scheduled workflow, do the pre-workflow things
+        if (wInfo.config.registration && (wInfo.config.registration as SchedulerRegistrationConfig).schedulerConfig) {
+          const inargs = args as unknown as [string, string, number, number]; // You would think this is a Date but it is a string.
```

Comment:
Would it be neater to define a type with these 4 properties? (Assuming we cannot reuse `ScheduledArgs` because these are now strings and not Dates)

Reply:
Generally having poor luck passing structs around.  Dates, at least, get resuscitated by the framework.